trashtalk/google_sheets.py

```diff
@@ -8,15 +8,17 @@
 from oauth2client.service_account import ServiceAccountCredentials
 
 from trashtalk.models import Cleanup, db_session
+from trashtalk.master_keys import MasterKeys
+key_chain = MasterKeys()
```

Comment:
The MasterKeys module isn't quite configured correctly. I haven't gotten around to fixing it so I disabled it in the meantime.

But there's also the question of how we're handling keys and currently we're using environment variables. I recommend we stick with that instead of doing this in two different places.

Did you have something else in mind for MasterKeys?